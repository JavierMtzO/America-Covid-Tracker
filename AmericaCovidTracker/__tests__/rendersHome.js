import React from 'react';
import Intro from '../Home';

const renderer = require('react-test-renderer');

test('Home renders correctly', () => {
    const tree = renderer.create(<Intro />).toJSON();
    expect(tree).toMatchSnapshot();
});