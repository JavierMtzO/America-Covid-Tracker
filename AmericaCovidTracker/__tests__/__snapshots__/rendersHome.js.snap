// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F46624",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 25,
        "paddingBottom": 100,
        "textAlign": "center",
      }
    }
  >
     Welcome to the America Covid Tracker
  </Text>
  <View
    accessibilityRole="button"
    accessibilityState={Object {}}
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Got to Map
      </Text>
    </View>
  </View>
</View>
`;
